README.md

```diff
@@ -0,0 +1,30 @@
+# Spectate Plugin for SourceMod
+
+Add a commands to spectate specific players and removes broken spectate mode.
+
+## Features
+
+- Enable or disable the plugin with a ConVar.
+- Limit the time a player can spend in spectate mode.
+- Control the spectate limit mode with a ConVar.
+- Force players to suicide before switching to spec.
+- Restrict the spectate list to admins only.
+- Integration with ZombieReloaded and EntWatch plugin.
+- Authorize users based on flags.
+- Limit the maximum time a player can spend in spectate mode.
+
+## ConVars
+
+- `g_cEnable`: Enable or disable the plugin.
+- `g_cSpecLimit`: Limit the time a player can spend in spectate mode.
```

Comment:
same thing

Reply:
I tried to generate the readme with Github Copilot 😄